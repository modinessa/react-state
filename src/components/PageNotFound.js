import React from "react";
import { Link } from "react-router-dom";


export function PageNotFound() {
	return (
		<div className="app-section">
			<div className="app-not-found">
				<h2>Page Not found</h2>
				<p>Looks like you've followed a broken link or entered a URL that doesn't exist on this site.</p>
				<Link to="/" className="link-not-found">
					&#10229; Back to our site
				</Link>
			</div>
		</div>
	)
} 