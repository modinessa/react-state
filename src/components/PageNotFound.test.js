import renderer from 'react-test-renderer';
import {BrowserRouter} from "react-router-dom"; 

import {PageNotFound} from './PageNotFound';

describe('PageNotFound', () => {
	it('renders component', () => {
		expect(renderer.create(
			<BrowserRouter>
				<PageNotFound/>
			</BrowserRouter>)).toMatchSnapshot();
	});
});