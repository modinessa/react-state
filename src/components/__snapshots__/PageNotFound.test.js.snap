// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound renders component 1`] = `
<div
  className="app-section"
>
  <div
    className="app-not-found"
  >
    <h2>
      Page Not found
    </h2>
    <p>
      Looks like you've followed a broken link or entered a URL that doesn't exist on this site.
    </p>
    <a
      className="link-not-found"
      href="/"
      onClick={[Function]}
    >
      ‚üµ Back to our site
    </a>
  </div>
</div>
`;
